merlin-server/sql/merlin/applied_migrations.sql

```diff
@@ -29,3 +29,4 @@ call migrations.mark_migration_applied('23');
 call migrations.mark_migration_applied('24');
 call migrations.mark_migration_applied('25');
 call migrations.mark_migration_applied('26');
+call migrations.mark_migration_applied('28');
```

Comment:
Keep an eye on this migration ID, if this PR gets merged before #1140

Reply:
Yeah, I was expecting the merge order to be 1140 -> this one, but it looks like I'll need to reverse the migration numbering 😔.